logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -469,3 +469,15 @@ private fun Conversation.Access.toDAO(): ConversationEntity.Access = when (this)
     Conversation.Access.LINK -> ConversationEntity.Access.LINK
     Conversation.Access.CODE -> ConversationEntity.Access.CODE
 }
+
+internal fun Conversation.Protocol.toApi(): ConvProtocol = when (this) {
+    Conversation.Protocol.PROTEUS -> ConvProtocol.PROTEUS
+    Conversation.Protocol.MIXED -> ConvProtocol.MIXED
+    Conversation.Protocol.MLS -> ConvProtocol.MLS
+}
+
+internal fun Conversation.Protocol.toDao(): Protocol = when (this) {
+    Conversation.Protocol.PROTEUS -> Protocol.PROTEUS
+    Conversation.Protocol.MIXED -> Protocol.PROTEUS
+    Conversation.Protocol.MLS -> Protocol.PROTEUS
```

Comment:
shouldn't be `Protocol.MLS` at leas for `Conversation.Protocol.MLS` ??

Reply:
Oops, copy paste error!

Reply:
Fixed